opentelemetry/proto/profiles/v1experimental/pprofextended.proto

```diff
@@ -123,6 +123,14 @@ message Profile {
   // Index into the string table of the type of the preferred sample
   // value. If unset, clients should default to the last sample value.
   int64 default_sample_type = 14;
+  // Documentation link for this profile. The URL must be absolute,
+  // e.g., http://pprof.example.com/cpu-profile.html
+  //
+  // The URL may be missing if the profile was generated by older code or code
+  // that did not bother to supply a link.
+  int64 doc_url = 19;  // Index into string table.
+
+  reserved 20 to max; // NEXT ID
```

Comment:
This is merely to make it clear what the next ID to use is. I.e. if a new field needs to be added it would be of ID = 20. I think this is orthogonal to renumbering the fields, but I'll remove it from this commit. After all I'm not sure if OTel has any recommendations on tracking / documenting the next ID in proto files. Sometimes people just use a comment for this, but the more recent way is to use this reserved trick.